docs/styleguide.md

```diff
@@ -43,13 +42,12 @@ Using `Quick Start` as example:
 For API references, there are exceptions to this rule.
 
 ## Markdown rules
-
+<!--TODO(erickzhao): make sure this matches with the lint:markdownlint task-->
 * Use `sh` instead of `cmd` in code blocks (due to the syntax highlighter).
-* Lines should be wrapped at 80 columns.
```

Comment:
IMO, folks should prefer to wrap at 80 or 100 columns for readability, but it's not necessary to enforce.

Reply:
Sure, I'll add that nuance in.